README.md

```diff
@@ -4,7 +4,7 @@ Copyright (C) 1998 - 2022 Daniel Stenberg, <daniel@haxx.se>, et al.
 SPDX-License-Identifier: curl
 -->
 
-# ![curl logo](https://curl.se/logo/curl-logo.svg)
+# <a href="https://curl.se/">![curl logo](https://curl.se/logo/curl-logo.svg)</a>
```

Comment:
please change this to use markdown format, for example refer to `OpenSSF Best Practices` below or any of the other images with link

Reply:
@jay thanks for your review. I made necessary changes. Looks good to me. Please review